### PR TITLE
Ensure that version of podman is upper than 4.4.2 before testing kube

### PR DESCRIPTION
**Pull request by ilausuch**
- Related ticket: https://progress.opensuse.org/issues/124601
VR:
- 15-SP4 http://openqa.suse.de/tests/10832660
- 15-SP3 http://openqa.suse.de/tests/10832455
- 15-SP2 http://openqa.suse.de/tests/10832662
- 15-SP1 http://openqa.suse.de/tests/10832661

### CODE REVIEW

File: lib/containers/utils.pm [hunk 1]
```diff
@@ -56,6 +56,13 @@ sub check_runtime_version {
     return check_version($other, $current, qr/\d{2}(?:\.\d+)/);
 }
 
+sub check_min_runtime_version() {
```
Comment (pablo-herranz) [inline lib/containers/utils.pm:59]: I'll choose a function name that explicitly said `podman`. Otherwise, LGTM

**Review by pablo-herranz — APPROVED** [review]

File: lib/containers/utils.pm (hunk 1 above)
Comment (ilausuch) [reply lib/containers/utils.pm:59]: its on utils, Check the previous function, I like the idea to have a checker for versions no mater what product

**Review by pdostal — APPROVED** [review]

File: tests/containers/podman_pods.pm [hunk 2]
```diff
@@ -42,7 +43,7 @@ sub run {
         assert_script_run('podman play kube --down hello-kubic.yaml');
     }
 
-    unless (is_sle('<=15-SP2')) {
+    unless (is_sle('<=15-SP2') || !check_min_runtime_version('4.4.2')) {
```
Comment (jlausuch) [inline tests/containers/podman_pods.pm:46]: maybe we can remove the check `is_sle('<=15-SP2')` completely, so we only depend on the podman version?

Comment (ilausuch) [reply tests/containers/podman_pods.pm:46]: ok

**Review by jlausuch — APPROVED** [review]

**Pull request merged**